runtime/moonbase/src/lib.rs

```diff
@@ -797,3 +817,107 @@ cumulus_pallet_parachain_system::register_validate_block!(
 	Runtime,
 	pallet_author_inherent::BlockExecutor<Runtime, Executive>
 );
+
+#[cfg(test)]
+mod multiplier_tests {
```

Comment:
These tests are quite verbose. I see that we have other test .rs files, would it make sense to add a new one specifically for these `multiplier_tests`?

Reply:
I let you judge on that

Reply:
Can we do these tests against the actual runtime in `tests/integration_test.rs` (instead of mocking a runtime with this feature enabled and then testing that mock runtime)?